### DISCUSSION TITLE
How to select parameters of sliding window inference

### DISCUSSION THREAD

**Discussion by muk465** [Project-MONAI/MONAI#4625] [Q&A]
I have 256*256*256 mri volume i am using random cropped patches  of size 128*128*128 for training my swinunetr /unetr 
with batch size of 1 ,ca anyone tell 
how do i select 
1)roi size
2)sw_batch size
3)overlap between patches
during testing?

**Comment by wyli** [comment]
Thanks, those are free system parameters, probably you can tune them with a grid search over a separate validation set.
